start.sh

```diff
@@ -27,17 +37,9 @@ if [ -f .env-saved ]; then
 
     if [[ ${REUSE_SAVED} =~ ^[Yy] ]]
     then
-        cat << EOI
-┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ Loading existing .env-saved file environment values...                                      ┃
-┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
-EOI
+      box_text -w 96 'Loading existing .env-saved file environment values...'
```

Comment:
I like some of the enhancements you've made, but calling `box_text` directly instead of `${OUTPUT} << EOI` breaks if PCRE-compatible `grep` is not available. Also, I suggest when this is complete, we back-port it to the social-app-template repo to keep in sync.

Reply:
it now calls $OUTPUT except where we're using box_text_attention, and that doesn't have any extra double-width characters being sent to it.